lib/test_prof/factory_bot.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module TestProf
+  CONSTANT_NAMES = { 'factory_bot' => '::FactoryBot', 'factory_girl' => '::FactoryGirl' }.freeze
```

Comment:
What about changing nam to `FACTORY_GIRL_NAMES` and using it in `FactoryCreate` too?